redaxo/src/addons/mediapool/pages/index.php

```diff
@@ -101,5 +101,15 @@
     <?php
 }
 
+// -------------- Minibar
+if (class_exists('rex_minibar')) {
```

Comment:
die minibar existiert immer. das ist nicht nötig

Reply:
Aber doch nur in REX 5.7? Dann müssten die beiden Core-AddOns `mediapool` und `structure` doch REX 5.7 als Min-Version bekommen, oder?

Reply:
Sie existiert nur immer, wenn wir die min-rex-version im medienpool entsprechend auf 5.7 setzen.

Reply:
hatte es dann nach abschicken auch bemerkt und den kommentar direkt oben noch geändert.

danke dennoch.

Reply:
No worries, passt. 👍